### ISSUE TITLE
Background dark in versions before a1.2.0, Also sound Proxy

### ISSUE THREAD

**Issue by Magallanesmapping321** [sojlabjoi/AlphacraftLauncher#1]
I dont know if its a bug or not but ive tried using a version that isnt b1.1_02 (a1.0.15 & a1.1.1), i get a darker background, unlike a1.2.0 versions or above which have the background lighter

And i recommend for nostalgia to have sound proxies for different versions, for example, a1.0.8 cow sounds instead of a1.0.9s cow sound

**Comment by sojlabjoi** [comment]
The sound proxy based on versions would be hard to implement as I haven't found a way to directly read the version string from the main JAR file. Unless, of course, I found a different way of determining the version.

I will be looking into both of the A1.1.1 and A1.0.15 and see what the issue may be. I'll get back to you.

**Comment by sojlabjoi** [comment]
I have found a solution for the background issue and have already implemented it.

**Comment by Magallanesmapping321** [comment]
> I have found a solution for the background issue and have already implemented it.

Thanks, it now works but i now type $MS and it just takes me to the cant connect page

**Comment by Magallanesmapping321** [comment]
Nevermind, i just forgot to place the password

**Comment by sojlabjoi** [comment]
After thinking, it seems that with how the launcher doesn't use the version instance system, adding specific sound proxies to specific versions is extremely difficult. An alternative would be to add some form of UI element, but that breaks the nostalgia. Besides, after some dirt digging, I found that back in the old days, you didn't get any old sounds back without having a backup of the old /resources/ folder in the working directory. Unfortunately, it can't be fixed, so I will have to close this comment as the previous issues have been fixed.
